opentech/static_src/src/app/src/containers/GroupByStatusDetailView.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux'
+
+import DisplayPanel from '@components/DisplayPanel';
+import DetailView from '@components/DetailView';
+import ByStatusListing from '@containers/ByStatusListing';
+import { setCurrentSubmissionRound } from '@actions/submissions';
+
+class GroupByStatusDetailView extends React.Component {
+    componentWillMount() {
+        this.props.setSubmissionRound(this.props.roundId);
```

Comment:
Perhaps we move this up to the SubmissionsByRoundApp - it has round in the name afterall...